draft-ietf-quic-transport.md

```diff
@@ -4330,6 +4374,12 @@ active_connection_id_limit (0x000e):
   to store. This value includes only connection IDs sent in NEW_CONNECTION_ID
   frames. If this parameter is absent, a default of 0 is assumed.
 
+alternate_version (0x000f):
+
+: The QUIC version to be used by the peer when sending 1-RTT long header packets
+  ({{packet-1rtt-long}}).  If this parameter is absent, the peer MUST NOT send
+  1-RTT long header packets.
```

Comment:
The concern here is that TP from the client is sent in clear.

I do not think that is a big issue in terms of greasing, but if we think that we need to address the problem, we can send the version number along with spare connection IDs. To paraphrase, adding "alternate_version" fields to preferred_address TP and NEW_CONNECTION_ID frame is an option.